modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java

```diff
@@ -289,6 +289,12 @@ private Map<String, String> _getAllParametersMap(
 			allParametersMap.put(key, String.valueOf(entry.getValue()));
 		}
 
+		if (!Objects.isNull(ddmDataProviderRequest.getLocale())) {
+			allParametersMap.put(
+				"ddm.locale",
```

Comment:
@p-albuquerque did you find this pattern (string starting with `ddm.`) elsewhere in the portal?

Reply:
I think it would be better to use `languageId` instead of `locale`. 

Perhaps something like `ddmDataProviderLanguageId`.

Reply:
I see some places where either data provider or form properties have similar prefix:
[here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/DDMStorageTypesDataProvider.java#L29), [here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/DDMDataProviderInstanceOutputParametersDataProvider.java#L39), [here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/DDMDataProviderInstancesDataProvider.java#L29). But I'm aware that those references isn't near to the context here, I really not so sure about my choice. I appreciate the suggestion!